Assets/Entities/Crawlers/Crawler.cs

```diff
@@ -71,7 +84,7 @@ void Start()
         }
         else
         {
-            sword.blade.GetComponent<Collider>().enabled = false;
+            //sword.blade.GetComponent<Collider>().enabled = false;
```

Comment:
Here the same damage will be registered by all clients, while it should be registered only by the server.

Reply:
Hmm... But `TakeHit` is run only on server anyways. Possibly, we can leave it like that.